src/TwigComponent/tests/Unit/EventListener/TwigComponentLoggerListenerTest.php

```diff
@@ -38,9 +38,9 @@ public function testLoggerStoreEvents(): void
         $eventB = new PreCreateForRenderEvent('b');
         $logger->onPreCreateForRender($eventB);
 
-        $eventC = new PreMountEvent(new \stdClass(), []);
+        $eventC = new PreMountEvent(new \stdClass(), [], new ComponentMetadata([]));
```

Comment:
Maybe one event with "null" ? As it's a nullable argument

Reply:
The argument is nullable only for BC purpose, I'm not sure that checking this BC fits the goal of this test. Maybe in another test?

Reply:
Yep you're right, let's focus :)